### PR TITLE
Use signin

### PR DESCRIPTION
**Pull request by carrieje**
This needs a [small fixup on cozy-konnector-libs](https://github.com/cozy/cozy-konnector-libs/pull/162).
It should not be merged until the patch is merged, and a new release is made.

### CODE REVIEW

**Comment by carrieje** [comment]
Here it is https://github.com/cozy/cozy-konnector-libs/pull/162

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "start": "node ./src/index.js",
     "dev": "cozy-konnector-dev build/index.js",
-    "standalone": "cozy-konnector-standalone build/index.js",
+    "standalone": "cozy-run-standalone build/index.js",
```
Comment (kosssi) [inline package.json]: peux tu supprimer `build/index.js` et juste laisser `"standalone": "cozy-run-standalone"` merci d'avance pareil pour la ligne du dessus ;)

Comment (carrieje) [reply package.json]: `cozy-run-dev` plutôt non ?
Je fais ça tout de suite ;-)

**Comment by carrieje** [comment]
3.8.1 has been released.
It works in local.

**Comment by ptbrowne** [comment]
Thanks Jeremy !

**Pull request merged**